mindsdb/integrations/handlers/shopify_handler/shopify_tables.py

```diff
@@ -398,6 +399,55 @@ def select(self, query: ast.Select) -> pd.DataFrame:
 
         return orders_df
 
+    def insert(self, query: ast.Insert) -> None:
+        """
+        Inserts data into the Shopify "POST /orders" API endpoint.
+
+        Parameters
+        ----------
+        query : ast.Insert
+            Given SQL INSERT query
+        
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        ValueError
+            If the query contains an unsupported condition
+        """
+        insert_statement_parser = INSERTQueryParser(
+            query,
+            supported_columns=['address1_ba', 'address2_ba', 'city_ba', 'company_ba', 'country_ba',
+                               'country_code_ba', 'first_name_ba', 'last_name_ba', 'latitude_ba', 
+                               'longitude_ba', 'name_ba', 'phone_ba', 'province_ba', 'province_code_ba', 
+                               'zip_ba', 
+                               'address1_sa', 'address2_sa', 'city_sa', 'company_sa',
+                               'country_sa', 'country_code_sa', 'first_name_sa', 'last_name_sa', 
+                               'latitude_sa', 'longitude_sa', 'name_sa', 'phone_sa', 'province_sa', 
+                               'province_code_sa', 'zip_sa', 
+                               'amount_dc', 'code_dc', 'type_dc',
+                               'gift_card_li', 'grams_li',  'price_li', 'quantity_li', 'title_li', 
+                               'vendor_li', 'fulfillment_status_li', 'sku_li', 'variant_title_li', 
+                               'name_li', 'value_li',
+                               'price_tl', 'rate_tl', 'title_tl', 'channel_liable_tl',
+                               'name_na', 'value_na',
+                               'code_sl', 'price_sl', 'discounted_price_sl', 'source_sl', 
+                               'title_sl', 
+                               'carrier_identifier_sl', 'requested_fulfillment_service_id_sl', 
+                               'is_removed_sl',
+                               'buyer_accepts_marketing', 'currency', 'email', 'financial_status', 
+                               'fulfillment_status', 'note', 'phone', 'po_number', 'processed_at', 
+                               'referring_site', 'source_name', 'source_identifier', 'source_url', 
+                               'tags', 'taxes_included', 'test', 'total_tax', 'total_weight'],
+            mandatory_columns=['price_li', 'title_li'],
+            all_mandatory=False
+        )
+        order_data = insert_statement_parser.parse_query()
+        self.create_orders(order_data)
```

Comment:
@aayushshah07 , that is a sound suggestion. I will aim to have this updated by the end of this weekend since I will be away from my computer for the next day or so. Out of curiosity, will you be handling future reviews of this pull request?

Reply:
Upon further inspection of the rest of the file (and a few other files from other handlers similar to this one), it is not standard to make functions analogous to the create_order function (e.g. update_orders function, delete_orders function) protected or private. I'd be interested to hear @MinuraPunchihewa 's thoughts on this. If functions like these should be protected or private it may be a good idea to track these changes in separate issues.